adam/curriculum/phase1_curriculum.py

```diff
@@ -2052,7 +2056,6 @@ def make_take_template(
                 ),
             )
         ],
-        constraining_relations=[bigger_than(agent, theme)],
```

Comment:
Hmm why did we have to remove the constraining relations from these templates?

Reply:
Since we are now filling templates with e.g. bear and cat, which have different relative size to the agents we're currently used to, we aren't able to learn anything unless we remove these.

Reply:
So e.g both `Mom` and a `bear` can be agents, and there are some objects smaller than `bear` but larger than `mom`? I thought the algorithm, once it matches the objects, only cares about the relations between them so this shouldn't matter? I might be misunderstanding the issue

Reply:
Right, I thought that the algorithm could handle this as well but these constraints cause the learning to completely break and removing them fixes it so I don't think it can.

Reply:
Okay! Can you make an issue for that?

Reply:
And I will merge once CI passes